ads/adition.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {writeScript, validateDataExists} from '../src/3p';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function adition(global, data) {
+
+  validateDataExists(data, ["version"]);
+
+  global.data = data;
+
+  writeScript(global, 'https://imagesrv.adition.com/js/amp/v'+String(data["version"])+'.js');
```

Comment:
Can you use `loadScript` instead?
Use `encodeURIComponent` on the version to avoid XSS.
Surround `+` with space and remove the `String` call (it does nothing here).